samples/zephyr/subsys/ipc/ipc_service/icmsg/Kconfig.sysbuild

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+source "share/sysbuild/Kconfig"
+
+config REMOTE_BOARD
+	string "The board used for remote target"
+	default "nrf54lm20pdk/nrf54lm20a/cpuflpr" if BOARD_NRF54LM20PDK_NRF54LM20A_CPUAPP
+	default "nrf54lv10dk/nrf54lv10a/cpuflpr" if BOARD_NRF54LV10DK_NRF54LV10A_CPUAPP
```

Comment:
Kconfigs should go and should use https://github.com/nrfconnect/sdk-nrf/blob/main/samples/benchmarks/coremark/sysbuild.cmake or optionally have CPU cluster here only then construct board target as in above link